tests/snapshots/test_graphql.test_graphql_error.json

```diff
@@ -0,0 +1,23 @@
+[[
+  {
+    "name": "graphql_sync",
```

Comment:
In Span.span.set_exc_info(), `Span.error = 1` is only set if a Span has a [traceback](https://github.com/DataDog/dd-trace-py/blob/v1.1.1/ddtrace/span.py#L426). 

`graphql.GraphqlError` does not generate a traceback for some errors. Ex: (Cannot query field)[https://github.com/graphql-python/graphql-core/blob/v3.2.1/src/graphql/validation/rules/fields_on_correct_type.py#L56]. 

Even though this Span does not have a traceback I would still expect it be marked as error Span. In cases like this should we generate a "fake" one line traceback or the current behavior acceptable?